scripts/get-job-matrix.py

```diff
@@ -143,6 +143,14 @@ class MakefileTest(TypedDict):
     "python": "3.8.x",
 }
 
+ALL_VERSION_SET = {
+    "name": ["all"],
+    "dotnet": ["6", "7", "8"],
```

Comment:
.NET 7 isn't supported.
.NET 6 will be unsupported tomorrow, so could prob leave it off

Reply:
Dropped 7. I left 6 for now, we should drop that when we drop it from the minimum version set as well.